binr/radiff2/radiff2.c

```diff
@@ -683,7 +683,7 @@ int main(int argc, char **argv) {
 	case MODE_DIFF:
 	case MODE_DIFF_STRS:
 	case MODE_DIFF_IMPORTS:
-		d = r_diff_new (0LL, 0LL);
+		d = r_diff_new ();
```

Comment:
yeah i know. it may only break in some platforms. but it's a reminder that we have to use (void) more in the .h